index.js

```diff
@@ -1,25 +1,175 @@
+/* eslint no-unused-vars: 0 */
+const projects = [
+  {
+    id: '1',
+    name: 'Tonic',
+    info: {
+      company_name: 'CANOPY',
+      position: 'Back End Dev',
+      date: '2015',
+    },
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: 'images/works_images/Snapshoot_Portfolio5.png',
+    technologies: ['html', 'javascript', 'css'],
+    live: '#',
+    source: '#',
+  },
+  {
+    id: '2',
+    name: 'Multi-Post Stories',
+    info: {
+      company_name: 'FACEBOOK',
+      position: 'Full Stack Dev',
+      date: '2015',
+    },
+    description:
+      'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    image: 'images/works_images/desktop_images/desktop1.png',
+    technologies: ['html', 'javascript', 'css', 'python'],
+    live: '#',
+    source: '#',
```

Comment:
[OPTIONAL] Why not make the project more realistic 🤔 by adding real links for these? Perhaps a link to your live portfolio for the live and a link to the GitHub repo for the source 🔗 . It can be the same link for all of them as these are dummy projects. 😉